.travis.yml

```diff
@@ -10,3 +18,8 @@ install: |
         export TRAVIS_BUILD_DIR="${MANTLE_DIR}"
         cd "${TRAVIS_BUILD_DIR}"
     fi
+
+script:
+  - ./test
```

Comment:
Not that it matters in this case because the two do not share artifacts but I just usually expect to see something built before tested :-P